.github/CODEOWNERS

```diff
@@ -1,3 +1,4 @@
 * @epoberezkin @efim-poberezkin
-/*Dockerfile @shumvgolove @epoberezkin @efim-poberezkin
+/Dockerfile @shumvgolove @epoberezkin @efim-poberezkin
 /scripts/docker/ @shumvgolove @epoberezkin @efim-poberezkin
+/scripts/main/ @shumvgolove @epoberezkin @efim-poberezkin
```

Comment:
The `scripts` directly doesn't have a `main` subdirectory in this repository yet.

Reply:
Same as above.